Week1-Coding-Task.hs

```diff
@@ -0,0 +1,76 @@
+--Section 1, credit card validation
+--Double value of every second digit from beginning of right 
+--Add digits of doubled and undoubled values to form new number 
+--Calculate result mod 10: if 0, valid 
+
+------------------------------------------------------------------------------
+--Exercise 1: toDigits and toDigitsRev 
+------------------------------------------------------------------------------
+toDigits :: Integer -> [Integer]
+toDigitsRev :: Integer -> [Integer]
+
+--A list with elements reversed, so can use it later for reversing toDigits 
+rev :: [Integer] -> [Integer]
+rev [] = []--base case 
+rev (s:l) = rev l ++ [s]--step case
```

Comment:
This is nice and short implementation. Though, there's exist better way to write `rev` function without `(++)` operator and using only `(:)`. It's more difficult but more efficient.